install/merger/types/config_map_client.go

```diff
@@ -0,0 +1,12 @@
+package types
```

Comment:
In general, in Go we place all related types within one file. It should rather be in configmap_override.go.